lib/models/addon.js

```diff
@@ -437,14 +437,15 @@ var Addon = CoreObject.extend({
     @example
     ```js
     treeForAddon: function(tree) {
+      tree = this._super.treeForAddon.call(this, tree);
```

Comment:
this should probably be `this._super.treeForAddon.apply(this, arguments);` instead to make sure it won't forget to pass any additional arguments